src/order/bounded_order.lean

```diff
@@ -1196,6 +1196,10 @@ lemma disjoint.comm : disjoint a b ↔ disjoint b a := by rw [disjoint, disjoint
 lemma symmetric_disjoint : symmetric (disjoint : α → α → Prop) := disjoint.symm
 lemma disjoint_assoc : disjoint (a ⊓ b) c ↔ disjoint a (b ⊓ c) :=
 by rw [disjoint, disjoint, inf_assoc]
+lemma disjoint_left_comm : disjoint a (b ⊓ c) ↔ disjoint b (a ⊓ c) :=
+by simp_rw [disjoint, inf_left_comm]
+lemma disjoint_right_comm : disjoint (a ⊓ b) c ↔ disjoint (a ⊓ c) b :=
+by simp_rw [disjoint, inf_right_comm]
```

Comment:
These lemma names feel very weird to me. Perhaps
```suggestion
lemma disjoint_inf_left_comm : disjoint a (b ⊓ c) ↔ disjoint b (a ⊓ c) :=
by simp_rw [disjoint, inf_left_comm]
lemma disjoint_inf_right_comm : disjoint (a ⊓ b) c ↔ disjoint (a ⊓ c) b :=
by simp_rw [disjoint, inf_right_comm]
```

Reply:
(and same for `codisjoint`)

Reply:
What do you think if I tell you they match [`disjoint_assoc`](https://leanprover-community.github.io/mathlib_docs/order/bounded_order.html#disjoint_assoc)?

Reply:
I'd begrudgingly be ok with your names, although ideally would suggest a follow-up to change all of them.

Reply:
I'll go forward with those and let you open the renaming PR if you feel like it :)